OmopTransformer/Transformation/GradeDifferentiationLookup.cs

```diff
@@ -8,7 +8,7 @@ internal class GradeDifferentiationLookup : ILookup
     public Dictionary<string, ValueWithNote> Mappings { get; } =
         new()
         {
-            { "GX", new ValueWithNote("4054711",    "GX grade") },
+            { "GX", new ValueWithNote("0",    "GX grade") },
```

Comment:
I have a suggestion. Take this out completely. Then update the XML and filter out any GX Grades.

The Data dictionary basically says that GX = "Grade of differentiation is not appropriate or cannot be assessed" which for me means a No Result, so no point adding it to OMOP at all